vite.config.ts

```diff
@@ -37,7 +37,7 @@ export default defineConfig({
         name: "ZinZen: Deliver purpose",
         icons: [
           {
-            src: "pwa-192x192.png",
+            src: "pwa-192x192-new.png",
```

Comment:
@vinaybadgujar102 Why rename the src if we're not using the old file anymore?

Reply:
resolved